spec/features/restaurant_spec.rb

```diff
@@ -68,6 +69,40 @@
   end
 end
 
+feature "viewer searches for a restaurant", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.